### DISCUSSION TITLE
[asciidoc] Diagram support for asciidoc content

### DISCUSSION THREAD

**Discussion by marcaurele** [community/community#21352] [Other Feature Feedback, Questions, & Ideas]
Hi!

I have a question regarding the support of Asciidoc, are you planning to add some diagrams rendering for file content written in asciidoc? So far from the test of everyone nothing is rendered and it outputs simply the content as code, which is a bit disappointing :disappointed: 

I see you're [adding some support for some admonitions on Markdown](https://github.com/community/community/discussions/16925), but this is all custom tweaking for GH flavored Markdown content.

I would have thought that Mermaid would be supported as it is currently available for Markdown.

**Comment by bric3** [comment]
Indeed that would be great to at least support mermaid for asciidoc content.
